x/oracle/genesis.go

```diff
@@ -29,8 +29,10 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, data types.GenesisState) {
 		k.SetWithdraw(ctx, withdraw)
 	}
 
-	for _, task := range tasks {
-		k.UpdateAndSetTask(ctx, task)
+	for i := range tasks {
+		task := tasks[i]
+		task.ClosingBlock = ctx.BlockHeight() + task.WaitingBlocks
```

Comment:
It was originally updated in `keeper.UpdateAndSetTask`. 
Now `UpdateAndSetTask` is refactored to take `TaskI`. So I guess it may be better to handle concrete-type persistent stuff before calling into the function.

Reply:
the code has been moves to keeper func and `ClosingBlock` was renamed to `ExpireHeight`.